api/src/wfl/module/all.clj

```diff
@@ -138,5 +141,19 @@
 (s/def ::name string?)
 (s/def ::methodConfiguration (s/and string? util/terra-namespaced-name?))
 (s/def ::methodConfigurationVersion integer?)
-(s/def ::watchers (s/* util/email-address?))
+(s/def ::email util/email-address?)
+(s/def ::legacyWatcher ::email)
+(s/def ::taggedWatcher (s/or :email slack/email-watcher?
+                             :slack slack/slack-channel-watcher?))
```

Comment:
This is OK, but naming these keys with the actual tags (`:EmailAddress`, for example) might make them easier to find and interpret in spec errors.

Reply:
Good idea, will change!